examples/kafka_tutorial.md

```diff
@@ -0,0 +1,67 @@
+# Kafka Labs
+
+In this tutorial we'll be walking through step-by-step instructions on getting a Kafka service up and running. This tutorial assumes you've already setup a local cluster using the [Quick Start](https://github.com/mesosphere/dcos-commons/blob/pods/README.md) guide.
+
+#### Steps:
+
+1. Create a new service by running `./new-service examples/kafka` in the dcos-commons root directory. This will create a clone of the `hello-world` service which you can then modify.
+2. `cd examples/kafka/src/main/dist/`
+3. With an editor of your choice open `svc.yml`. Notice that some values, such as `<task-name>_COUNT` and `<task-name>_CPUS` are mustached.[0]
+4. Since Kafka has Brokers, change the type of the task in the Kafka pod from `server` to `broker`.
+  1. Also notice that `count` is specified as a property of the `kafka` pod. This indicates there will be `count` kafka pods, each of whom will run a `broker`.
+5. Next we'll open up the necessary port. Add the following as a property to the `broker` task:
```

Comment:
Suggest: "Next, add the following property to the `broker` task in order to open up the necessary port."